types/src/types.ts

```diff
@@ -1542,3 +1542,17 @@ export type A2AError =
   | InvalidAgentResponseError
   | AuthenticatedExtendedCardNotConfiguredError;
 // --8<-- [end:A2AError]
+
+// --8<-- [start:JSONError]
+/**
+ * Represents the error format for HTTP+JSON/REST transport error responses.
+ */
+export interface JSONError {
+  /** A2A-specific error identifier (e.g., "TASK_NOT_FOUND"). */
+  code: string;
+  /** Human-readable error description. */
+  message: string;
+  /** Optional additional structured error information. */
+  details?: any;
+}
+// --8<-- [end:JSONError]
```

Comment:
Not sure if I missed this on my earlier review. Is there a reason to have another type here in addition to the `A2AError`? Should `JSONError` be an extension of or contain an `A2AError` instance?